ts/screens/services/__tests__/ServiceDetailsScreen.test.tsx

```diff
@@ -35,6 +38,25 @@ describe("ServiceDetailsScreen", () => {
         component.getByText(service.organization_fiscal_code)
       ).toBeDefined();
     });
+
+    it("should render the ogranization's fiscal code", () => {
+      const { component, store } = renderComponent();
+      store.dispatch(
+        loadServiceDetail.failure({
+          error: new Error("load failed"),
+          service_id: service.service_id
+        })
+      );
+      expect(
+        component.getByText(service.organization_fiscal_code)
+      ).toBeDefined();
+    });
+  });
+
+  it("expect to match snapshot", () => {
+    const { component } = renderComponent();
+
+    expect(component).toMatchSnapshot();
```

Comment:
This is a whole screen and indeed the snapshot ts/screens/services/__tests__/__snapshots__/ServiceDetailsScreen.test.tsx.snap is some ~1300k lines long.
No one will ever bother to check it in the next diff making the test more of a hindrance than a resource.
I suggest removing it and stick to DOM assertions for large components.